experience.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Godzilla for Hire | Experience/Interests</title>
+  <link rel="stylesheet" type="text/css" href="css/styles.css">
+  <link href="https://fonts.googleapis.com/css?family=Bowlby+One+SC|Signika" rel="stylesheet">
+</head>
+<body class="body">
+  <header class="header">
+    <nav class="header__nav">
+      <a class="nav__links" href="index.html">Bio</a> | 
+      <a class="nav__links__current" href="#">Experience/Interests</a>
+    </nav>
+  </header>
+  <main class="main">
+    <h1 class="main__h1">Experience/Interests</h1>
+    <h2 class="main__h2">Cities Attacked/Destroyed</h2>
+    <ul class="main__unordered">
+      <li class="main__listitem">Tokyo, Japan</li>
+      <li class="main__listitem">Osaka, Japan</li>
+      <li class="main__listitem">Beijing, China</li>
+      <li class="main__listitem">Las Vegas, NV, USA</li>
+      <li class="main__listitem">Honolulu, HI, USA</li>
+      <li class="main__listitem">San Francisco, CA, USA</li>
+      <li class="main__listitem">New York City, NY, USA</li>
+    </ul>
+    <h2 class="main__h2">Monsters Fought</h2>
+    <ul class="main__unordered">
+      <li class="main__listitem">Anguiras</li>
+      <li class="main__listitem">King Ghidorah</li>
+      <li class="main__listitem">King Kong</li>
+      <li class="main__listitem">Mechagodzilla</li>
+      <li class="main__listitem">Mecha King Ghidorah</li>
+      <li class="main__listitem">Rodan</li>
+      <li class="main__listitem">Gigan</li>
+      <li class="main__listitem">Megalon</li>
+      <li class="main__listitem">Kamacuras</li>
+      <li class="main__listitem">Kumonga</li>
+      <li class="main__listitem">Hedorah</li>
+      <li class="main__listitem">A Giant Vulture</li>
+      <li class="main__listitem">Titanosaurus</li>
+      <li class="main__listitem">Gabara</li>
+      <li class="main__listitem">Mothra</li>
+      <li class="main__listitem">Battra</li>
+      <li class="main__listitem">MOGUERA</li>
+      <li class="main__listitem">Biollante</li>
+      <li class="main__listitem">Space Godzilla</li>
+      <li class="main__listitem">Destroyah</li>
+      <li class="main__listitem">Orga</li>
+      <li class="main__listitem">Meganulon/Megaguiras</li>
+    </ul>
+    <h2 class="main__h2">Current Working Groups</h2>
+    <ul class="main__unordered">
+      <li class="main__listitem">Strategies for Kaiju Understanding and Human Destruction (SKUHD)</li>
+      <li class="main__listitem">Forum on Innovation and Feasibility of Destroying Tokyo (FIFDT)</li>
+    </ul>
+  <div class="full-width">
+    <img class="wide-image" src="assets/godzilla_3.jpg" alt="" />
```

Comment:
Although not required, the best practice is to include the `alt` attribute which allows for accessibility to the visually impaired and is what appears if the resource fails to load.